aioseop_class.php

```diff
@@ -4502,7 +4502,7 @@ function wp_head() {
 		if ( ! apply_filters( 'aioseop_disable_schema', false ) ) {
 			// Handle Schema.
 			if ( version_compare( PHP_VERSION, '5.5', '>=' ) ) {
-				if ( ! empty( $aioseop_options['aiosp_schema_markup'] ) && boolval( $aioseop_options['aiosp_schema_markup'] ) ) {
```

Comment:
Should be fine. I believe Michael wants to keep this as is so that when we slowly drop support for older versions we can simply switch to using newer PHP features like boolval() for example.